osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -111,11 +109,22 @@ public void Start()
             });
         }
 
+        /// <summary>
+        /// When <see cref="Start"/> is called, this will be run to give an opportunity to prepare the clock at the correct
+        /// start location.
+        /// </summary>
+        protected virtual void PrepareStart()
+        {
+            // Seeking the decoupled clock to its current time ensures that its source clock will be seeked to the same time
+            // This accounts for the clock source potentially taking time to enter a completely stopped state
+            Seek(GameplayClock.CurrentTime);
```

Comment:
As per my comment above, this is now pretty much pointless since the clock is always processed regardless of pause state.

Reply:
I've removed this for now.